packages/grafana-eslint-rules/rules/no-border-radius-literal.cjs

```diff
@@ -0,0 +1,48 @@
+// @ts-check
+const { ESLintUtils, AST_NODE_TYPES } = require('@typescript-eslint/utils');
+
+const createRule = ESLintUtils.RuleCreator((name) => `https://github.com/grafana/grafana#${name}`);
+
+const borderRadiusRule = createRule({
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          node.callee.type === AST_NODE_TYPES.Identifier &&
+          node.callee.name === 'css' &&
+          node.arguments[0].type === 'ObjectExpression'
+        ) {
+          for (let property of node.arguments[0].properties) {
+            if (
+              property.type === AST_NODE_TYPES.Property &&
+              property.key.type === AST_NODE_TYPES.Identifier &&
+              property.key.name === 'borderRadius' &&
+              property.value.type === AST_NODE_TYPES.Literal
+            ) {
+              context.report({
+                node: node.arguments[0],
```

Comment:
@eledobleefe if we set this to `property`, the error highlighting is a bit more specific (to the actual line, rather then the whole `css({})` declaration
```suggestion
                node: property
```